server.js

```diff
@@ -21,9 +15,117 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+    {"Welcome to the ultimate BOOK REVIEW LIST!":  [
+{
+      "books": "All books available", 
+      "/books/id/:id": "Search for a book by specific id",
+      "/books/title/:title": "Search for a book by specific title",
+      "/books/authors/:authors": "Search for book/books written by specific author (alone or with others)",
+      "/books/language_code/:language_code": "Search for books available in specific language",
+    }
+    ]
+  } );
 });
```

Comment:
I really enjoyed your "welcome" page! Really nice of you to specify each endpoint you have created. Considering there is no frontend on this project, I would say this is the most user friendly thing to do :) Nice and easily read from my perspective!